backend/src/infrastructure/LoggingExtensions.cs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty file?

Reply:
I have moved the file to the folder, don't know why an empty file took it place 🤷🏼  probably I closed all tabs in editor and it saved it. Sorry, I have not noticed